### DISCUSSION TITLE
Compliance Date Verson Enforcemement Date for TIC

### DISCUSSION THREAD

**Discussion by ericsilvertx** [CMSgov/price-transparency-guide#217] [General]
We all know that CMS will start enforcement on June 2022.

As for your own solution are you still tracking to be done by Jan 2021?

I had a chat this week and I have that only about 10% of the Health Plans will be ready by Jan 2022?

What Date are you tracking to be done?  Jan 2022 or June 2022

**Comment by shaselton-usds** [comment]
@ericsilvertx Sorry for the delayed response. Enforcement starts July 1st.

CMS plans to have v1.0, the version in which will be considered final for compliance, on March 1st.
